ImagePicker/CellRegistrator.swift

```diff
@@ -160,19 +114,32 @@ public final class CellRegistrator {
         assetItemClassesData?[type] = (cellClass, cellIdentifier)
     }
     
-    ///
     /// Register a cell class for all asset items types (image and video).
-    ///
     public func registerCellClassForAssetItems<T: UICollectionViewCell>(_ cellClass: T.Type) where T: ImagePickerAssetCell {
         assetItemClass = cellClass
     }
     
-    ///
     /// Register a cell nib for all asset items types (image and video).
     ///
     /// Please note that cell's class must conform to `ImagePickerAssetCell` protocol, otherwise an exception will be thrown.
-    ///
     public func registerNibForAssetItems(_ nib: UINib) {
         assetItemNib = nib
     }
+    
+    func cellIdentifier(forActionItemAt index: Int) -> String? {
+        // First lets check if there is a registered cell at specified index
+        if let index = actionItemNibsData?[index]?.1 ?? actionItemClassesData?[index]?.1 {
+            return index
+        }
+        
+        // If not found globaly registered return nil
+        guard index < Int.max else { return nil }
+        
+        // Lets see if there is a globally registered cell for all indexes
+        return cellIdentifier(forActionItemAt: Int.max)
+    }
+    
+    func cellIdentifier(forAsset type: PHAssetMediaType) -> String? {
+        return assetItemNibsData?[type]?.1 ?? assetItemClassesData?[type]?.1
```

Comment:
it's not really readable in such format. Can you provide a more readable form of syntax ?

Reply:
changed .1 to identifier